### PR TITLE
Update Capital Framework and use family-linked webfonts

### PR DESCRIPTION
**Pull request by Scotchester**
This PR will eliminate the use of the old `u-webfont-` mixins, replacing them with the standard `font-style` and `font-weight` CSS properties that are now able to be used with the included upgrade to the latest version of cf-typography.

## Removals

- Outdated styles
- PDFreactor fonts that are no longer needed

## Changes

- Converted `u-webfont-` mixin calls to `font-style` or `font-weight` properties

## Testing

1. Pull branch
1. `npm install`
1. `gulp`
1. Browse through your local site and observe no changes from the production site

## Todos

- [x] Before this gets merged, we need another release of CF and to increment the versions updated here.

## Checklist

* [ ] PR has an informative and human-readable title
* [ ] Changes are limited to a single goal (no scope creep)
* [ ] Code can be automatically merged (no conflicts)
* [ ] Code follows the standards laid out in the [development playbook](https://github.com/cfpb/development)
* [ ] Passes all existing automated tests
* [ ] Any *change* in functionality is tested
* [ ] New functions are documented (with a description, list of inputs, and expected output)
* [ ] Placeholder code is flagged
* [ ] Visually tested in supported browsers and devices
* [ ] Project documentation has been updated
* [ ] Reviewers requested with the [Reviewer tool](https://help.github.com/articles/about-pull-request-reviews/) :arrow_right:

### CODE REVIEW

**Comment by Scotchester** [comment]
Completed the to-do, rebased on `master`, and force-pushed.

File: cfgov/unprocessed/css/organisms/featured-content-module.less [hunk 1]
```diff
@@ -1,105 +0,0 @@
-// TODO:
```
Comment (jimmynotjim) [inline cfgov/unprocessed/css/organisms/featured-content-module.less]: Thank you for taking care of this, I completely forgot.

File: cfgov/unprocessed/css/video-player.less [hunk 2]
```diff
@@ -227,3 +227,19 @@
 	   text-align: right;
 	}
 } );
+
+
+// Special styles for playing within an FCM
+.o-featured-content-module {
+  &_visual {
```
Comment (jimmynotjim) [inline cfgov/unprocessed/css/video-player.less]: Minor: Spaces should be 4, not 2

Comment (Scotchester) [reply cfgov/unprocessed/css/video-player.less]: Addressed in 2c8bc3e, rebased from master, force pushed.

**Comment by sebworks** [comment]
Would is still make sense to have a variable or utility for setting the fonts? This way you are able to standardize the particular font weights.

**Comment by Scotchester** [comment]
> Would is still make sense to have a variable or utility for setting the fonts? This way you are able to standardize the particular font weights.

I'm not totally sure I follow you, @sebworks. Do you mean that if we did something like `font-weight: @medium;`, we could change them all at once if we ever decided to, say, shift our "medium" weight from 500 to 600 and our "bold" weight from 600 to 700?

**Comment by sebworks** [comment]
>> I'm not totally sure I follow you, @sebworks. Do you mean that if we did something like font-weight: @medium;, we could change them all at once if we ever decided to, say, shift our "medium" weight from 500 to 600 and our "bold" weight from 600 to 700?

Yeah

**Comment by cfarm** [comment]
I'm going to update cf-core with the [latest release](https://github.com/cfpb/capital-framework/pull/756) to get the demibold fixes in this branch.

**Comment by Scotchester** [comment]
Rebased and awaiting Travis!

**Comment by Scotchester** [comment]
@sebworks That's not a bad idea, but I'd like to put it on the backlog so I can go ahead and merge this to help Cat see some things through before she's out next week.

**Pull request merged**